app/assets/javascripts/search.coffee

```diff
@@ -43,19 +43,20 @@ $(document).ready ->
     switch recordType
       when 'variables'
         $searchButtonSpan.text('Search Variables').addClass('variable-text')
+      when 'services'
+        $searchButtonSpan.text('Search Services').addClass('service-text')
       else
-        $searchButtonSpan.text('Search Collections').removeClass('variable-text')
+        $searchButtonSpan.text('Search Collections')
```

Comment:
should we have a class for collections also, i.e. `collection-text`

Reply:
the reason there is a variable-text class is to keep the button the same width as it is when 'Search Collections is visible, I missed adding in the css for service-text, so I'll add in collection-text as well